pages/admin/content-type-builder/[entity_type_slug].js

```diff
@@ -47,218 +51,299 @@ export default function ContentTypeBuilder({cache}) {
     isLoading: true,
   };
 
-  const SET_SHOW = 'SET_SHOW';
-  const SET_ATTRIBUTES = 'SET_ATTRIBUTES';
-  const SET_ENTITY_TYPE_NAME = 'SET_ENTITY_TYPE_NAME';
-  const SHOW_DELETE_CONFIRMATION_MODAL = 'SHOW_DELETE_CONFIRMATION_MODAL';
-  const HIDE_DELETE_CONFIRMATION_MODAL = 'HIDE_DELETE_CONFIRMATION_MODAL';
+  const SET_SHOW = "SET_SHOW";
+  const SET_ATTRIBUTES = "SET_ATTRIBUTES";
+  const SET_ENTITY_TYPE_NAME = "SET_ENTITY_TYPE_NAME";
+  const SHOW_DELETE_CONFIRMATION_MODAL = "SHOW_DELETE_CONFIRMATION_MODAL";
+  const HIDE_DELETE_CONFIRMATION_MODAL = "HIDE_DELETE_CONFIRMATION_MODAL";
+  const SHOW_EDIT_CTYPE_MODAL = "SHOW_EDIT_CTYPE_MODAL";
+  const HIDE_EDIT_CTYPE_MODAL = "HIDE_EDIT_CTYPE_MODAL";
 
-  const LOADING = 'LOADING';
+  const LOADING = "LOADING";
 
   const reducer = (state, action) => {
-    
-    switch(action.type) {
-        case SET_SHOW:
-          return {
-            ...state,
-            show: action.payload
-            }
-              
-        case SET_ATTRIBUTES:
-          return {
+    switch (action.type) {
+      case SET_SHOW:
+        return {
+          ...state,
+          show: action.payload,
+        };
+
+      case SET_ATTRIBUTES:
+        return {
+          ...state,
+          attributes: action.payload,
+        };
+
+      case SET_ENTITY_TYPE_NAME:
+        return {
+          ...state,
+          entity_type_name: action.payload,
+        };
+
+      case LOADING:
+        return {
+          ...state,
+          isLoading: action.payload,
+        };
+
+      case SHOW_DELETE_CONFIRMATION_MODAL:
+        return {
+          ...state,
+          showDeleteConfirmationModal: true,
+        };
+
+      case HIDE_DELETE_CONFIRMATION_MODAL:
+        return {
           ...state,
-          attributes: action.payload
-          }
-
-        case SET_ENTITY_TYPE_NAME:
-          return {
-            ...state,
-            entity_type_name: action.payload
-          }
-
-        case LOADING: 
-          return {
-            ...state,
-            isLoading: action.payload
-          }
-
-        case SHOW_DELETE_CONFIRMATION_MODAL:
-          return {
-            ...state,
-            showDeleteConfirmationModal: true
-          }
-
-        case HIDE_DELETE_CONFIRMATION_MODAL:
-          return {
-            ...state,
-            showDeleteConfirmationModal: false
-          }
+          showDeleteConfirmationModal: false,
+        };
+
+      case SHOW_EDIT_CTYPE_MODAL:
```

Comment:
As we are not limited to the number of characters, can we spell our constants out, so that future developers will not be confused?
```suggestion
      case SHOW_EDIT_CONTENT_TYPE_MODAL:
```